src/components/modes/teacher/Settings.js

```diff
@@ -301,6 +344,12 @@ class Settings extends Component {
           </InputLabel>
           {selectControl}
         </FormControl>
+        <FormControl>
```

Comment:
Also, add some left margin (maybe `theme.spacing(2)`) so that the drop down menus are not touching.

Reply:
![image](https://user-images.githubusercontent.com/45897168/88455849-a4404580-ce81-11ea-890b-4073915b8dfe.png)